src/client.js

```diff
@@ -143,6 +143,10 @@ assign(Client.prototype, {
 
     return this._query(connection, obj).catch((err) => {
       err.message = this._formatQuery(obj.sql, obj.bindings) + ' - ' + err.message
+      let {detail} = err
+      if (detail) {
+        err.message += '\nDetail: ' + detail
```

Comment:
can you add test for that?

Reply:
sure I can. Do you require a functional one or will unit test be enough?

Reply:
I think that unit test should be perfectly sufficient in this case.